news/2018-09-19-the-qat-gazette-september-2018.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: "The QAT Gazette: September 2018"
+date: 2018-10-19 16:00:00 +0000
+---
+
+Huge changes to Beatmap Nominator applications, updates on improvements to Modding Discussions, and a brand new blog contributor? What will they come up with next? This month’s issue of the QAT Gazette has all the answers!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+A new month brings new adventure, new challenges, and a new face to our humble blog!
+
+A lot has happened in the past month and as you might expect, you can find it all right here! From substantial changes to the way Beatmap Nominator applications are conducted, to ongoing developments within the Modding Discussions system and of course, some aforementioned new faces, keep reading to find out everything you need to know in and around the mapping and modding community.
```

Comment:
```suggestion
A lot has happened in the past month, and as you might expect, you can find it all right here! From substantial changes to the way Beatmap Nominator applications are conducted, to ongoing developments within the Modding Discussions system and of course, some aforementioned new faces, keep reading to find out everything you need to know in and around the mapping and modding community.
```

Reply:
The first two clauses are not independent, so technically they shouldn't be linked with a comma if joined by "and". If it's about speech pausing, there's already one after "expect" so I don't think another is necessary here.